charter.html

```diff
@@ -233,44 +237,41 @@ <h2 id=proposals>Proposals</h2>
 
-<p>The <a href=#chairs>Chairs</a> may create a dedicated repository for
-a Proposal. They may do this for any reason, such as but not limited to:
+<p>Champions are responsible for the technical content of their
+Proposal. They are expected to solicit input from Community Group
+Participants, and they may consider and respond to comments,
+suggestions, and objections on their Proposal from Participants and the
+public.
 
-<ul>
-<li>because the Proposal has become sufficiently complex that writing an
-explainer is warranted
-<li>because the Proposal is sufficiently complex that having a dedicated
-issue tracker for it is warranted
-<li>because multiple <a href=#implementer>implementers</a> have
-expressed interest in the Proposal
-</ul>
+<p>Champions may ask the <a href=#chairs>Chairs</a> to create a
+dedicated repository for their Proposal.
 
 <p>Proposals begin as or evolve into
 <a href=https://w3ctag.github.io/explainers>explainers</a> which
-describe the proposed changes to the web platform, and which may serve
-as the basis for a Specification (should the Proposal be adopted as
-one).
-
-<p class=note>While explainers are Community Group Reports as defined in
-the <a href=https://www.w3.org/community/about/agreements/>Community and
-Business Group Process</a>, they are not Specifications as defined in
-that document.
+describe a user-facing problem which needs to be solved and how the
+authors propose to solve it. Explainers are Community Group Reports as
+defined in the
+<a href=https://www.w3.org/community/about/agreements/>Community and
+Business Group Process</a>, but they are not Specifications as defined
+in that document.
 
-<p>Proposals may be withdrawn by their originators, or may be closed by
+<p>Proposals may be withdrawn by their Champions, or may be closed by
 the Chairs (if, for example, the Chairs deem the Proposal to
-be <a href=out-of-scope>out of scope</a> or the Proposal fails to gain
-sufficient <a href=#implementer>implementer</a> support to be adopted as
-a Specification). If such a Proposal has a dedicated repository, the
-Chairs should take steps to ensure the data is not lost, perhaps by
+be <a href=out-of-scope>out of scope</a>, or if its number of Champions
+drops to zero). If such a Proposal has a dedicated repository, the
+Chairs and Champions should take steps to ensure the data is not lost,
+perhaps by
 transferring the repository to a different organization or user, or by
 archiving it.
 
 <div id=work-items hidden><!-- keep old links working --></div>
 <h2 id=specifications>Specifications</h2>
 
-<p>The Community Group may produce <dfn>Specifications</dfn>—as defined
-in the <a href=https://www.w3.org/community/about/agreements/>Community
-and Business Group Process</a>—a special kind of Community Group Report
+<p>Per the
+<a href=https://www.w3.org/community/about/agreements/>Community and
+Business Group Process</a>, this Community Group may
+produce <dfn>Specifications</dfn>—a special kind of Community
```

Comment:
I don't object to this, I just want to capture that "Specification" is a general-purpose term, and things will definitely be referred to as "specifications" even when they're not adopted by the CG.  As long as you're not concerned about this, I'm fine with it; otherwise, I'd choose a more unique-to-this-CG term, like Work Item (which I think you used before?) or the like.

Reply:
Yeah, I think there are reasonable arguments for and against using a bespoke term. That said, since these things are Specifications as defined by the CBGP, we may as well call them that. (This is the rationale behind #23.)